### ISSUE TITLE
the dimension of inference result was 3

### ISSUE THREAD

**Issue by KARAMKIM88** [fizyr/keras-maskrcnn#96]
I ran your sample code(ResNet50MaskRCNN.ipynb)

I didn't fix any code

I thought it worked fine

but the result didn't have mask information.

**Comment by hgaiser** [comment]
What is the result then?

**Comment by ujjawalcse** [comment]
How to get mask information @KARAMKIM88 @hgaiser ?
I'm also getting only 3 items in result i.e bounding_box,score and labels.
How to fix this ?

**Comment by KARAMKIM88** [comment]
I didnt find the solution
So i used pytorch and detectron
Sorry

2020년 3월 3일 (화) 오후 8:07, Ujjawal Kumar Singh <notifications@github.com>님이
작성:

**Comment by ujjawalcse** [comment]
Okay Thanks.
